density/density.py

```diff
@@ -329,6 +329,13 @@ def capacity():
         cap_data = json.load(json_data)['data']
     # Read current data
     cur_data = db.get_latest_data(g.cursor)
+    locations = calculate_capacity(cap_data, cur_data)
+    return render_template('capacity.html', locations=locations)
+
+#Calculates capacity with cap_data and cur_data and puts with respective
```

Comment:
These comments should be moved to a [docstring](https://www.python.org/dev/peps/pep-0257/) in the lines after the `def`.
See the other functions defined in this file for an example